app/retail/emails.py

```diff
@@ -304,44 +304,59 @@ def render_new_bounty_roundup(to_email):
     subject = "Open Source Money, Open Source Jobs "
 
     intro = '''
+        <p>Hi there</p>
```

Comment:
We should probably maintain the same indentation we had on this block to ensure we're not adding unnecessary whitespace to the output.

Reply:
@mbeacom so remove the whole indentation?